CoreScriptsRoot/Modules/Settings/SettingsHub.lua

```diff
@@ -272,7 +297,61 @@ local function CreateSettingsHub()
 			Parent = this.Shield
 		};
 
-		local barHeight = 60
+		--Bumper hints
+		this.BumperHintBar = utility:Create("Frame") {
+			Parent = this.Shield,
+			Visible = false,
+			Name = "BumperHints",
+			BackgroundTransparency = 1,
+
+			ZIndex = SETTINGS_BASE_ZINDEX + 1,
+
+			Position = UDim2.new(0, 0, 0, 0),
+			Size = UDim2.new(1, 0, 0, 64)
+		}
+	
+		this.BumperHintLeft = utility:Create("ImageLabel") {
+			Parent = this.BumperHintBar,
+			Visible = true,
+			Name = "BumperHintLeft",
+			Image = "rbxassetid://408462759",
```

Comment:
We should have a local asset(we will probably need it in a lot of places and it is better than downloading it all the time)